rrmngmnt/host.py

```diff
@@ -490,3 +491,62 @@ def is_connective(self, tcp_timeout=20.0):
             "Use Host.executor().is_connective() instead."
         )
         return self.executor().is_connective(tcp_timeout=tcp_timeout)
+
+    def set_iptables(
```

Comment:
this is operation very specific, and I think it might be widely extended in the future.
I would do one of following things:
* move this method to 'network' module
* create 'firewall' module where you can keep all firewall related methods